0.0.903

AlfaVlados opened the pull request:
Добавление цвета пуль.
Цвет пуль соответствует цвету стреляющего

AlfaVlados committed:
0.0.903

Добавление цвета пуль.
Цвет пуль соответствует цвету стреляющего

Chuchumbrik commented on Rouge Light Project/Assets/Scripts/Shooting/Shooting.cs:
```diff
@@ -1,83 +1,198 @@
+    {
+        // ���������, �������� �� ������� ������ ������� (����� ��� "Player")
+        // ��� �����, ��� ��� ���������� ������������� ������ ����� ���������� �� ������
+        if (gameObject.CompareTag("Player"))
```
Я бы попробовал иначе, к примеру, у тебя есть gameobject - это тот кто стреляет
Проверяем, есть ли у него SpriteRenderer отдельный
Если да, то берем цвет из него
Если нет, то берем SpriteRenderer InChildren и берем цвет из него
если не то и не то, то вызываем ошибку и дефолтный какой нибудь вычурный цвет

AlfaVlados committed:
0.0.904

Исправлен алгоритм для присваивания цвета пули (Shooting.cs)
Добавлена окантовка для пули (Bullet.cs)
